### PR TITLE
Adds endpoint info for saml Authorization.

### PR DESCRIPTION
**Pull request by flamingbear**
API documentation for new SAML endpoints.

### CODE REVIEW

File: content/samlAuth.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+x-amz-apigw-id: BDaHGF19oAMFt_g=
+vary: Accept
+location: https://auth.launchpad-sbx.nasa.gov/affwebservices/public/saml2sso?SAMLRequest=pZLNbtswEIRfReBdP4wkWyZkBW6MogbS1ojdHHopVtTKJiCRqpa03T59ablBc2kuuS5m95sZsry%2F9F1wwpGU0UvGo4TdV%2BXK2aN%2Bwp8OyQZeoGnJ3KiFAVIkNPRIwkqxW31%2BFHdRIobRWCNNx25iQdB3b28AEY7WM1nw%2FAL3cxZs1kv2o0WOWToDjpjNmzSV85QvZkWa46zJ0zZpinTRZOm84H6ByOFGkwVt%2FY2EL0KehEm25wuR5yL3kXjynQVrn0VpsBPqaO1AIo7BB406cFoeB2hCqi%2BRBoLoYE4xtO0Za2%2FzpCRSPLi6UzK%2BRrsjMixYvUR4MJpcj%2BPuJv329PgP4AqN8%2FqgZBPhBaWzGMKgIkchAtmQR9DDb6PhTJE0vSiSJIsbPE2YyR0Ltn%2FL%2FaB0o%2FTh7V7rm4jEp%2F1%2BG26%2F7vYs%2BGhGidOTLlkLHSGryut9MVU3Vu%2F1Wsavr5VfvKnNemt8W7%2Bu7B7s%2Fz3ziE8T1YTtJBVO04BStQob33HXmfPDiGBxyezokMVVGb%2F%2BntUf&RelayState=http%3A%2F%2Flocalhost%3A3000%2F%23%2Fauth
+x-powered-by: Express
+X-Amzn-Trace-Id: Root=1-5d97a3c6-19bd637148b511a5ec3e1850;Sampled=0
+x-amzn-Remapped-date: Fri, 04 Oct 2019 19:55:51 GMT
+```
+
+The example response shows a redirect to the Identity provider SAML2 Single Sign On page with an encoded request.  Following the location header will take you to a modal login page for Launchpad (or similiar SAML provider based on the Cumulus configuration).
```
Comment (kkelly51) [inline content/samlAuth.md]: Are we sure we even want to put /saml/login in the api docs? 
It isn't meant to be hit by curl at all. Only the dashboard. So I don't know if I want to encourage people to try.

File: content/samlAuth.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Successful authorization will redirect the SAML workflow back to the Assertion Consumer Service at `/saml/auth`.
+
+## authorize
+
+```endpoint
+POST /saml/auth
+```
+
+The `saml/auth` endpoint recieves and validates an HTTP POST from a SAML Identity Provider after a successful login.
```
Comment (kkelly51) [inline content/samlAuth.md]: Same with this. 
These I think are real only for SAML interactions. I wasn't planning on putting them in the API docs.

File: content/samlAuth.md (hunk 1 above)
Comment (flamingbear) [reply content/samlAuth.md]: I don't know.  I just assumed we should document each endpoint, but I def have no strong opinions.

File: content/samlAuth.md (hunk 2 above)
Comment (flamingbear) [reply content/samlAuth.md]: So do you think there is anything we should put in the docs about the saml endpoints?

**Comment by flamingbear** [comment]
Closing.  Don't want to encourage people to use the end points this way. Will add documentation elsewhere to describe the configuration and use of the endpoints.

**Pull request closed without merging**